internal/workflow/preprocessing.go

```diff
@@ -550,20 +550,22 @@ func writePREMISFile(ctx temporalsdk_workflow.Context, sip sip.SIP) error {
 	}
 
 	// Add PREMIS events for validate file activity.
+	veraPDFAgent := premis.Agent{
+		Type:    "software",
+		Name:    "VeraPDF 1.26.2",
```

Comment:
I think we need to make this configurable or read the version from the system. For now, I'd add it to the `[filevalidate.verapdf]` configuration as `premisAgentName`:

https://github.com/artefactual-sdps/preprocessing-sfa/blob/main/hack/kube/overlays/dev/preprocessing-secret.yaml#L59-L60

Reply:
I see, we already had this hardcoded, so I'll merge as it is and make a note in the issue, thanks @mcantelon!